reflex/utils/exec.py

```diff
@@ -225,41 +255,77 @@ def run_backend_prod(
         if not config.gunicorn_workers
         else config.gunicorn_workers
     )
-    RUN_BACKEND_PROD = f"gunicorn --worker-class {config.gunicorn_worker_class} --preload --timeout {config.timeout} --log-level critical".split()
-    RUN_BACKEND_PROD_WINDOWS = f"uvicorn --timeout-keep-alive {config.timeout}".split()
     app_module = f"reflex.app_module_for_backend:{constants.CompileVars.APP}"
-    command = (
-        [
-            *RUN_BACKEND_PROD_WINDOWS,
-            "--host",
-            host,
-            "--port",
-            str(port),
-            app_module,
-        ]
-        if constants.IS_WINDOWS
-        else [
-            *RUN_BACKEND_PROD,
-            "--bind",
-            f"{host}:{port}",
-            "--threads",
+
+    if should_use_granian():
+        try:
```

Comment:
Should we do this in a more modular way? Rather than having if statements here make it possible to runners from the outside.

Reply:
For now I want people to be able to test it to get feedback on it, since it'll only be used if the flag is set.
We can make it more modular in the future if feedback is positive, to let people choose what they want.